resources/store/mutations.js

```diff
@@ -48,7 +48,14 @@ module.exports = {
 		} else {
 			state.images[ state.currentTab ].push( payload.image );
 		}
+	},
 
+	/**
+	 * @param {Object} state
+	 * @param {Object} suggestion
+	 */
+	addSuggestionToCurrentImage: function ( state, suggestion ) {
+		state.images[ state.currentTab ][ 0 ].suggestions.push( suggestion );
```

Comment:
In mutations, why are we not using the `currentImage` getter? I noticed it here and in `toggleSuggestion`. Is that a vuex no-no?

Reply:
Vuex mutations don't actually have access to `getters` when they are called, only to state. Which makes sense because a getter returns derived state, like a computed property. We shouldn't be using getters to actually mutate the underlying state.

Unfortunately in some cases it means we have to deal with a long chain of accessors to get what we want, but the only alternative would be to re-work the store so that things are structured differently.

Reply:
Got it, thanks!